public/controllers/agent/components/register-agent.js

```diff
@@ -569,16 +588,24 @@ export const RegisterAgent = withErrorBoundary(
             <Fragment>
               <EuiSpacer />
               <EuiText>
-                <EuiCodeBlock style={codeBlock} language={language}>
-                  {this.systemSelector()}
-                </EuiCodeBlock>
-                <EuiCopy textToCopy={this.systemSelector()}>
-                  {(copy) => (
-                    <EuiButton fill iconType="copy" onClick={copy}>
-                      Copy command
-                    </EuiButton>
-                  )}
-                </EuiCopy>
+                <div className="copy-codeblock-wrapper">
+                  <EuiCodeBlock style={codeBlock} language={language}>
+                    {this.systemSelector()}
+                  </EuiCodeBlock>
+                  <EuiCopy textToCopy={this.systemSelector()}>
+                    {(copy) => (
+                      <div className="copy-overlay" onClick={copy}>
+                        <p><EuiIcon type="copy" /> Copy command</p>
+                      </div>
+                    )}
+                  </EuiCopy>
+                </div>
+                <EuiSwitch
+                  label="Show password"
+                  checked={this.state.showPassword}
+                  onChange={(active) => this.setShowPassword(active)}
+                />
+                <EuiSpacer />
```

Comment:
**thought**: I am not sure why a switch to change the visibility of password is displayed here. The code block displays the code to enable and start the `wazuh-agent` service.